### PR TITLE
Network importer updated for diffsync 1.2.0

### PR DESCRIPTION
**Pull request by carbonarok**
Small changes to models and adapters to handle ObjectNotFound errors.

### CODE REVIEW

**Review by glennmatthews — APPROVED** [review]
LGTM, thanks!

File: tests/unit/adapters/network_importer/test_load_batfish_ipaddress.py [hunk 1]
```diff
@@ -46,8 +48,16 @@ def test_load_batfish_ipaddress_wo_ip_wo_prefix(network_importer_base, site_sfo,
     prefix = Prefix(prefix="10.10.10.0/24", site_name="sfo")
 
     assert isinstance(ipaddr, IPAddress)
-    assert adapter.get(IPAddress, identifier=ipaddr.get_unique_id()) is None
-    assert adapter.get(Prefix, identifier=prefix.get_unique_id()) is None
+    try:
+        res = adapter.get(IPAddress, identifier=ipaddr.get_unique_id())
+    except ObjectNotFound:
+        res = None
+    assert res is None
```
Comment (glennmatthews) [inline tests/unit/adapters/network_importer/test_load_batfish_ipaddress.py:55]: Simpler pattern for all of these test changes:

```python
with pytest.raises(ObjectNotFound):
    adapter.get(IPAddress, identifier=ipaddr.get_unique_id())
```

Comment (carbonarok) [reply tests/unit/adapters/network_importer/test_load_batfish_ipaddress.py]: Changes added

**Review by glennmatthews — APPROVED** [review]

**Review by dgarros — APPROVED** [review]
LGTM

**Pull request merged**